frontend/src/bundles/landing/components/subscription-part/subscription-part.tsx

```diff
@@ -15,15 +15,25 @@ const DEFAULT_INPUT: { email: string } = {
 };
 
 const SubscriptionPart: React.FC<Properties> = ({ title }) => {
-    const { control, errors } = useAppForm<{ email: string }>({
+    const { control, errors, reset } = useAppForm<{ email: string }>({
         defaultValues: DEFAULT_INPUT,
     });
 
+    const handleFormSubmit = useCallback(
```

Comment:
i think we can use notification service with message. Smt like "You're subscribed successfully!"

Reply:
done